source/_docs/scripts/conditions.markdown

```diff
@@ -220,29 +220,37 @@ Within an automation, template conditions also have access to the `trigger` vari
 
 ### Time condition
 
-The time condition can test if it is after a specified time, before a specified time or if it is a certain day of the week.
+The time condition can test if it is after a specified time or before a specified time.
 
 ```yaml
 condition:
   condition: time
   # At least one of the following is required.
   after: '15:00:00'
   before: '02:00:00'
-  weekday:
-    - mon
-    - wed
-    - fri
 ```
 
-Valid values for `weekday` are `mon`, `tue`, `wed`, `thu`, `fri`, `sat`, `sun`.
 Note that if only `before` key is used, the condition will be `true` *from midnight* until the specified time.
 If only `after` key is used, the condition will be `true` from the specified time *until midnight*.
 Time condition windows can span across the midnight threshold if **both** `after` and `before` keys are used. In the example above, the condition window is from 3pm to 2am.
 
-<div class='note tip'>
+### Weekday condition
 
-A better weekday condition could be by using the [Workday Binary Sensor](/integrations/workday/).
+The weekday condition can test if it is a certain day of the week.
 
+```yaml
+condition:
+  condition: weekday
+  days:
+    - mon
+    - wed
+    - fri
+```
+
+Valid values for `days` are `mon`, `tue`, `wed`, `thu`, `fri`, `sat`, `sun`.
+
+<div class='note tip'>
```

Comment:
We don't have a tip message box.
Let's remove the whole message box, leaving just the contents as normal.

Reply:
![image](https://user-images.githubusercontent.com/1741838/77540177-e7be7f00-6ea2-11ea-8ca3-f7f9e1eb04ce.png)
this is from docs page: https://www.home-assistant.io/docs/scripts/conditions/#time-condition

Reply:
Sure, but `tip` still is invalid and not defined.

Reply:
I've removed the `tip` class. As I wrote I looked at the current version.
I can remove `tip` class in other places (across `_docs` folder) in a separate PR if needed.